include/seqan3/alignment/matrix/detail/score_matrix_single_column.hpp

```diff
@@ -46,12 +48,15 @@ namespace seqan3::detail
  * view is the seqan3::detail::affine_cell_proxy, which offers a practical interface to access the value of the
  * optimal, horizontal and vertical value of the underlying matrices.
  */
-template <arithmetic score_t>
+template <typename score_t>
+//!\cond
+    requires arithmetic<score_t> || simd_concept<score_t>
```

Comment:
```suggestion
    requires (arithmetic<score_t> || simd_concept<score_t>)
```

Reply:
I don't understand why the paranthesis are necessary here. This should be converted to the disjunctive normalform or not?

Reply:
Just for the record: You are right, here it would not be necessary. (we talked about this on the phone)